### DISCUSSION TITLE
[Supression] 23. Le poids des images.md

### DISCUSSION THREAD

**Discussion by florinesueur** [cnumr/best-practices-wordpress#104] [Bonnes pratiques] [answered]
@hrenaud et @yannkozon, il me semble que vous traitez du même sujet dans les deux BP suivantes :
- https://github.com/cnumr/fiches-wordpress-greenit/blob/develop/best-practices-WordPress/fiches/22.%20Utiliser%20une%20version%20d'image%20ayant%20taille%20coh%C3%A9rente%20avec%20son%20utilisation.md
- https://github.com/cnumr/fiches-wordpress-greenit/blob/develop/best-practices-WordPress/fiches/23.%20Le%20poids%20des%20images.md

**Comment by hrenaud** [comment]
C'est que les sujets se sont rapprochés, mais initialement, ce n'était pas cas.
- 22 -> Mettre en place un système de vignettes, faire que WP génère des tailles d'images suivant le contexte, ce qu'aime PageSpeed.
- 23 -> C'était travailler la compression (Photoshop avec réduction de qualité avant upload et/oumettre en place un plugin de compression, etc.).

Ce qui est pour moi deux sujets différents (comme le type d'image qui en est un 3e).

**Comment by florinesueur** [comment, marked as answer]
Du coup comme la BP a dévié, je propose que la 23 devienne : Compresser les médias (vectoriel, matriciel, vidéo, audio).
Code : FFMPEG and co.
Logiciels : Photoshop and co.
No code : Extensions in Wordpress

Vous en pensez quoi ?
